database_sanitizer/config.py

```diff
@@ -93,7 +93,7 @@ def load_addon_packages(self, config_data):
             )
 
         for index, module_path in enumerate(section_addons):
-            if not isinstance(module_path, six.text_type):
+            if not isinstance(module_path, six.string_types):
```

Comment:
Do you really want to support both binary and Unicode strings here?

Reply:
Well not sure really. I could for example have:
```python
if isinstance(module_path, six.string_types):
    module_path = six.text_type(module_path)
```
Would that make more sense?

Reply:
Looked into it again and found out that `PyYAML` support custom yaml tags which solves this issue altogether. By using `!!python/unicode` tag we can already specify that the config file value should be converted into Python unicode object.

Changed README to include this tip.

Reply:
Module paths should always be string type, i.e. `str`. That's the type Python uses when storing the module name in `sys.modules` and therefore it won't make any sense to force this type to be `unicode` in Python 2.

So rather than `isinstance(module_path, six.text_type)` this should be `isinstance(module_path, str)`.

Reply:
And also this should be `isinstance(..., str)`: https://github.com/andersinno/python-database-sanitizer/blob/v0.4.1/database_sanitizer/config.py#L140